### DISCUSSION TITLE
Problem in resolving .local

### DISCUSSION THREAD

**Discussion by shanbs** [hassio-addons/addon-ssh#497] [Q&A] [answered]
Hi,

I am running a HassOS on a Raspberry Pi 4, and recently installed WireGuard add-on, followed the basic configuration from the documentation, and set the DNS server to be 172.30.32.3.

Everything works well with the exception of resolving .local addresses.

It can resolve `homeassistant`, but cannot resolve `homeassistant.local`. However, the `nslookup` can resolve .local addresses:

```
$ nslookup
> homeassistant.local.
Server:		172.30.32.3
Address:	172.30.32.3#53

Name:	homeassistant.local
Address: 172.30.32.1
Name:	homeassistant.local
Address: 192.168.1.121
Name:	homeassistant.local
Address: 172.30.232.1
```

Thanks!

**Comment by frenck** [comment, marked as answer]
`.local` = mDNS, not DNS.

../Frenck

**Reply by shanbs** [reply]
Thanks Frenck! Any suggestion about how to resolve the mDNS addresses?

**Reply by frenck** [reply]
That won't work unless you are going to broadcast multicast traffic across VPN connections (which I would strongly recommend not doing).

**Reply by shanbs** [reply]
Roger that. Thanks!
